src/databricks/labs/ucx/cli.py

```diff
@@ -470,6 +470,19 @@ def migrate_tables(w: WorkspaceClient, prompts: Prompts, *, ctx: WorkspaceContex
             deployed_workflows.run_workflow("migrate-external-tables-ctas")
 
 
+@ucx.command
+def migrate_acls(w: WorkspaceClient, prompts: Prompts, *, ctx: WorkspaceContext | None = None, **named_parameters):
+    """
+    Trigger the migrate-tables workflow and, optionally, the migrate-external-hiveserde-tables-in-place-experimental
+    workflow and migrate-external-tables-ctas.
+    """
+    if ctx is None:
+        ctx = WorkspaceContext(w)
+    ctx.acl_migrator.migrate_acls(
+        target_catalog=named_parameters.get("targe_catalog"), legacy_table_acl=True, principal=True
```

Comment:
typo. introduce a variable.

also declare a flag in `labs.yml`. provide a screenshot of successfully run command.

Reply:
Recorded and sent!!!